src/pages/groupComparison/pathwayMapper/GroupComparisonPathwayMapper.tsx

```diff
@@ -50,6 +60,7 @@ export default class GroupComparisonPathwayMapper extends React.Component<
         | undefined = undefined;
     @observable isGeneSelectionPopupVisible: boolean = false;
     @observable private _userSelectedGenes: string[] | undefined = undefined;
+    @observable currentPathway: string = '';
```

Comment:
I don't think we need this observable anymore, let's remove it.

We should also remove `clearMessage: (() => void) | undefined;` We don't update it anywhere.